scheduled/redshift.test.js

```diff
@@ -60,6 +64,8 @@ describe('Redshift lambda', () => {
         expect(error).toBeNull()
         expect(message).toBeDefined()
         expect(message).toEqual('Redshift deltas successful.')
+        expect(pgSuccessfulQuery).toHaveBeenCalledWith('DELETE FROM scale_of_belief.events\n      USING staging_events\n      WHERE scale_of_belief.events.id = staging_events.id')
```

Comment:
Since you're in here, do you think it's worth adding a line for the scores query as well?